app/Repositories/UserRepository.php

```diff
@@ -58,24 +79,32 @@ public function checkfriend($id, $customer_id)
         return $query->count();
     }
 
+    /**
+     * display profile mine
+     * @param integer $id
+     * @return \App\Models\User
+     */
     public function profile($id)
     {
-        return $this->model->select('users.*')
-                    ->where('users.id', '=', $id)->get();
+        return $this->model->select('users.*')->where('users.id', '=', $id)->get();
```

Comment:
Mặc định của laravel eloquent là select * nên câu query chỉ cần như sau là đủ
return $this->model->where('id', '=', $id)->get();